docs/pages/develop/development-builds/expo-go-to-dev-build.mdx

```diff
@@ -0,0 +1,116 @@
+---
+title: Convert from Expo Go to a development build
+sidebar_title: Expo Go to development build
+description: How to migrate your Expo Go project to use development builds
+---
+
+import { ContentSpotlight } from '~/ui/components/ContentSpotlight';
+import { Terminal } from '~/ui/components/Snippet';
+import { Step } from '~/ui/components/Step';
+
+To migrate from Expo Go to a development build, you'll need to:
+
+<Step label="1">
+
+## Install the `expo-dev-client`
+
+The Expo Dev Client library includes the launcher UI (shown in the screenshots below), dev menu, extensions to test updates, and more. The Expo Go app has the dev menu built in, and that's why you need to install it separately for a development build.
+
+<Terminal cmd={['$ npx expo install expo-dev-client']} />
+
+<ContentSpotlight
+  alt="expo-dev-client launcher and menu UIs on Android and iOS."
+  src="/static/images/dev-client/preview.png"
+  caption={`When you run a development build it will look like this, only with your app name and icon included rather than "Microfoam". The launcher UI is pictured in iOS on the left and Android on the right. In between, you can see an app running inside of the development build, with the customizable developer menu open.`}
+/>
+
+> We recommend using the `expo-dev-client` for the best development experience, but it is possible to use development builds without installing this library. If not using the dev client, in [Step 3](/develop/development-builds/expo-go-to-dev-build/#start-the-dev-client) start the bundler with `--dev-client`. Otherwise it will default to opening in Expo Go.
+
+</Step>
+
+<Step label="2">
+
+## Build your native app
+
+With Expo Go, we only needed to build the JavaScript bundle, but with development builds we also need to compile the native app. With Expo, there is two parts to building your native app:
+
+1. generate the native `/ios` and/or `/android` directories ([read more](/develop/development-builds/expo-go-to-dev-build/#cng-and-prebuild) on when and how you'll need to do this)
+2. use native build tools to compile the native app(s)
+
+Once you've built your native app, you won't need to build it again unless you add or update a library with native code, or change any native coding such as the app name.
+
+> After generating your `/ios` and/or `/android` directories, make sure you add them to `.gitignore` instead of checking them into Git. This ensures you can always regenerate the code locally or on CI when needed and never have to edit native code manually.
+
+### Option 1: Build on your local machine
+
+To build a native app on your local machine, follow the guide on [setting up your local development environment](https://reactnative.dev/docs/set-up-your-environment) for which ever platform you're looking to build for. This involves setting up and configuring native build tools like Xcode for iOS, and Android Studio for Android.
```

Comment:
```suggestion
To build a native app on your local machine, follow the setup your environment guides for [Android](/workflow/android-studio-emulator/) and [iOS](/workflow/ios-simulator/) platforms. This involves setting up and configuring native build tools like Android Studio for Android and Xcode for iOS.
```

Let's link to the guides from our docs instead of React Native instead of React Native doc.